kg_microbe/transform_utils/bacdive/bacdive.py

```diff
@@ -1221,179 +1294,41 @@ def run(self, data_file: Union[Optional[Path], Optional[str]] = None, show_statu
-                        #                         BACDIVE_PREFIX + key,
-                        #                         EXACT_MATCH,
-                        #                         BACDIVE_PREFIX + key,
-                        #                     ],
-                        #                     [
-                        #                         curated_strain_ids[j],
-                        #                         SAME_AS_PREDICATE,
-                        #                         BACDIVE_PREFIX + key,
-                        #                         EXACT_MATCH,
-                        #                         BACDIVE_PREFIX + key,
-                        #                     ],
-                        #                 ]
-                        #             )
-                    # ! ----------------------------
-
-                    if ncbitaxon_id and medium_ids:
+                    # Medium edges - link to strain organism_id, create NCBITaxon node if needed
+                    if medium_ids:
                         for mid, mlabel in zip(medium_ids, medium_labels, strict=False):
-                            # Combine list creation and extension for nodes
+                            # Create nodes for medium and NCBITaxon (if available)
                             nodes_data_to_write = [
-                                [ncbitaxon_id, NCBI_CATEGORY, ncbi_label],
                                 [mid, MEDIUM_CATEGORY, mlabel],
                             ]
+                            if ncbitaxon_id:
+                                nodes_data_to_write.append([ncbitaxon_id, NCBI_CATEGORY, ncbi_label])
+
                             nodes_data_to_write = [
                                 sublist + [None] * (len(self.node_header) - 3)
                                 for sublist in nodes_data_to_write
                             ]
                             node_writer.writerows(nodes_data_to_write)
 
-                            # Combine list creation and extension for edges
-                            edges_data_to_write = [
-                                [
-                                    organism,
-                                    NCBI_TO_MEDIUM_EDGE,
-                                    mid,
-                                    IS_GROWN_IN,
-                                    BACDIVE_PREFIX + key,
-                                ]
-                                for organism in species_with_strains
-                            ]
-
-                            edge_writer.writerows(edges_data_to_write)
+                            # Create edge(s) from organism(s) to medium
+                            for target in feature_targets:
+                                edge_writer.writerow(
+                                    [
+                                        target,
+                                        NCBI_TO_MEDIUM_EDGE,
```

Comment:
I think @turbomam diagreed with the use of this predicate (`biolink:occurs_in`) but we can use it for this release and make an issue to find a better predicate for the next release.